app/src/main/java/org/dosomething/letsdothis/ui/IntroActivity.java

```diff
@@ -1,63 +1,127 @@
 package org.dosomething.letsdothis.ui;
+import android.content.Context;
+import android.content.Intent;
 import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
-import android.view.View;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.ActionBarActivity;
+
+import com.viewpagerindicator.CirclePageIndicator;
 
 import org.dosomething.letsdothis.R;
-import org.dosomething.letsdothis.utils.AppPrefs;
+import org.dosomething.letsdothis.ui.fragments.BaseIntroFragment;
+import org.dosomething.letsdothis.ui.fragments.IntroFragment;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Created by toidiu on 4/15/15.
  */
-public class IntroActivity extends AppCompatActivity
+public class IntroActivity extends ActionBarActivity implements BaseIntroFragment.PagerChangeListener
```

Comment:
TODO: ActionBarActivity is deprecated, we should be extending AppCompatActivity.